PWGHF/TableProducer/candidateSelectorDsToKKPi.cxx

```diff
@@ -105,7 +125,8 @@ struct HfCandidateSelectorDsToKKPi {
         hfMlResponse.setModelPathsLocal(onnxFileNames);
       }
       hfMlResponse.init();
-      outputMl.assign(((std::vector<int>)cutDirMl).size(), -1.f); // dummy value for ML output
+      outputMlDsToKKPi.assign(((std::vector<int>)cutDirMl).size(), -1.f); // dummy value for ML output
+      outputMlDsToPiKK.assign(((std::vector<int>)cutDirMl).size(), -1.f); // dummy value for ML output
```

Comment:
```suggestion
      outputMlDsToKKPi.assign( static_cast<std::vector<int>>(cutDirMl).size(), -1.f); // dummy value for ML output
      outputMlDsToPiKK.assign( static_cast<std::vector<int>>(cutDirMl).size(), -1.f); // dummy value for ML output
```

Reply:
I suggest you how to write the same thing as you should do for a c+++ type-cast. However, can you test locally if this type-cast is necessary or not?

Reply:
Hi @mfaggin, thanks for your suggestions. I have tested locally and without the type cast I got the error: `error: 'struct o2::framework::Configurable<std::vector<int> >' has no member named 'size'`.

Reply:
Did you try `cutDirMl.value.size()`?

Reply:
No, I didn't try it. I have tried it now and it works. Thanks @vkucera!